src/RFC-0313_VNRegistration.md

```diff
@@ -0,0 +1,138 @@
+
+## Intro
+The way [Cerberus](https://www.radixdlt.com/post/cerberus-infographic-series-chapter-i) works, it's imperative that the list of active Validator Nodes (VN)s are publicly known. 
+Luckily we have the [Base Layer] where we can publicly commit to VNs. Because we have this list of active VNs, we can use the list when determining [VNC]s.
+
+## Registration
+Each VN wishing to participate in the DAN layer must publish a registration UTXO to the [Base Layer]. This registration UTXO MUST be flagged
+as a registration UTXO type. A UTXO on the [Base Layer] allows all nodes, including pruned nodes, to verify the list of active VNs. 
+
+We want to make VN registration Sybill-resistant, and to enforce this, the registration UTXOs MUST have at least a value of `VN registration amount` amount of Tari locked up. 
+This amount is determined by consensus code and can be only be changed with Hard-Forks. This amount should be high enough that the user who owns the
+registration UTXO will de-register the VN to reclaim the amount.
+
+Because we cannot force VNs who have published registration UTXOs to be active and participate in the DAN layer, we can incentivize them by allowing them to reclaim their registration UTXOs as a form of 
+de-registration. We cannot stop stale UTXOs, but we can ignore them by enforcing VNs to re-register every `VN re-registration` period of blocks. If [base node]s via consensus
+remove VNs whose registration UTXOs are older than the `VN re-registration` period, we can limit this. 
+
+> NOTE: An aggressive approach may be to force an anyone can spend script on the UTXO if the UTXO goes stale.
+
+In order to shuffle VNs inside of their respective VNCs, VNs MUST get a new VN-key each time they register. This is done to prevent some
+operators to do shard targeting attacks. 
+
+Each registration UTXO MUST contain the VN public key with a signature proving it knows the private key. 
+The signature MUST as a schnorr type signature as follows:
+$$
+\begin{aligned}
+sig = r + e*k \\\\
+e = hash(R || K || Commitment) \\\\
+\end{aligned}
+$$
+As the challenge, we hash the nonce, VN public key, and the commitment. We include the commitment to ensure each signature must be unique. This is important
+as we don't want users to reuse all signatures. 
+
+
+## VN-Key
+
+The VN-key is the unique key used to identify where in the [shard space] this key is generated unique each time the VN is registered to assign the
+VN to a different area of the [shard space] each time.
+
+The key MUST be calculated as follows:
+
+$$
+\begin{aligned}
+\text{VN-key} = hash(\text{VN-pubkey}||\text{blockhash})
+\end{aligned}
+$$
+Where the `blockhash` is the blockhash of the block, the registration transaction was mined in.
+It is essential we ensure that the `VN-Key` has some random entropy inside of it, as the `VN-Key` needs to be pseudorandom.
+
+## VN-Key option two
```

Comment:
I don't see the need for this option. The first option is much easier to understand